examples/csv-payments/self-host/container/demo-client.py

```
@@ -0,0 +1,305 @@
+CSV_FOLDER_TYPE = "org.pipelineframework.csv.grpc.PipelineTypes$CsvFolder"
+
+
+def request(method, url, token=None, body=None, timeout=10):
+    data = None
+    headers = {"Accept": "application/json"}
+    if body is not None:
+        data = json.dumps(body).encode("utf-8")
+        headers["Content-Type"] = "application/json"
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+    req = urllib.request.Request(url, data=data, headers=headers, method=method)
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)